src/CTA.WebForms2Blazor/ControlConverters/ButtonControlConverter.cs

```diff
@@ -9,19 +10,20 @@ public class ButtonControlConverter : ControlConverter
         protected override Dictionary<string, string> AttributeMap { 
             get
             {
-                throw new NotImplementedException();
+                return new Dictionary<string, string>()
+                {
+                    ["onclick"] = "@onclick", 
+                    ["cssclass"] = "class"
+                };
             } 
         }
-        protected override string BlazorName { get { throw new NotImplementedException(); } }
-        
-        public ButtonControlConverter() : base()
-        {
-            
-        }
+        protected override string BlazorName { get { return "button"; } }
 
         public override HtmlNode Convert2Blazor(HtmlNode node)
         {
-            throw new NotImplementedException();
+            var textAttr = node.Attributes.AttributesWithName("text").Single();
+            var buttonText = textAttr.Value;
```

Comment:
I'm not sure, but a label without any text would seem like unexpected behavior in the first place. I'll add a try catch to handle that

Reply:
Instead of adding a try catch, consider this instead:
```csharp
var textAttr = node.Attributes.AttributesWithName("text").FirstOrDefault();
var buttonText = textAttr?.Value ?? string.Empty;
```
This sets a default value of string.Empty if the text attribute is null. I'd expect this to be an edge case, but better to handle it if we can.

^ this suggestion also applies to my other comment